modules/ROOT/partials/configuration/force_hex_color.adoc

```diff
@@ -0,0 +1,21 @@
+[[force_hex_color]]
+== `+force_hex_color+`
+
+This option allows the integrator to optionally force the conversion of either RGB or HEX colors.
```

Comment:
Always will convert both rgb and rgba to hex. `rgb_only` will only convert `rgb` values but keep rgba value intact.

```suggestion
This option allows the integrator to optionally force the conversion of either RGB or RGBA colors into HEX colors.
```